docs/MAINTAINING.md

```diff
@@ -215,6 +215,18 @@ We publish code-server as a npm package [here](https://www.npmjs.com/package/cod
 
 This is currently automated with the release process.
 
+## Syncing with Upstream VS Code
+
+The VS Code portion of code-server lives under [`cdr/vscode`](https://github.com/cdr/vscode). To update VS Code for code-server, follow these steps:
+
+1. `git checkout -b vscode-update` - Create a new branch locally based off `main`
```

Comment:
❓ Is this the best approach?

Originally @TeffenEllis suggested going to the GitHub UI and hitting "fetch upstream." From a dev's perspective, that makes it super easy! The downside to that approach is there may be merge conflicts to resolve locally. If there are, they can't be reviewed by another developer.

Thoughts @code-asher?

Reply:
I like this approach.  We will likely keep having conflicts for a while (we already currently have conflicts with upstream).